src/scripts/sync_subclassof.py

```diff
@@ -0,0 +1,103 @@
+"""Create a robot template for purpose of syncing subclassof relations.
+
+Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/92
+- GitHub PR: https://github.com/monarch-initiative/mondo-ingest/pull/363
+"""
+import os
+from argparse import ArgumentParser
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Tuple, Union
+
+import pandas as pd
+from oaklib import get_adapter
+from oaklib.interfaces.basic_ontology_interface import BasicOntologyInterface, RELATIONSHIP
+from oaklib.types import CURIE, URI
+
+HERE = Path(os.path.abspath(os.path.dirname(__file__)))
+SRC_DIR = HERE.parent
+ONTOLOGY_DIR = SRC_DIR / 'ontology'
+DEFAULT_OUTPATH = ONTOLOGY_DIR / 'reports' / 'sync-subclassof.robot.template.tsv'
+DEFAULT_ONTOLOGY_DB_PATH = ONTOLOGY_DIR / 'tmp' / 'mondo-ingest.db'
+ROBOT_ROW = [{
+    'term_id': 'ID',
+    'parent_id': 'SC %',
+    'source_ids': '> A oboInOwl:source SPLIT=|',
+}]
+
+
+def sync_subclassof(
```

Comment:
I added details on the approach here: https://github.com/monarch-initiative/mondo-ingest/issues/92#issuecomment-1741773769

I noticed that we never really specified exactly what needed to be done. And there are still some holes in my conceptual model, I hope though I provide now sufficient detail to get this whole thing going.